monkey/monkey_island/cc/server_setup.py

```diff
@@ -133,12 +135,14 @@ def _get_deployment() -> Deployment:
 
 def _initialize_di_container(
     ip_addresses: Sequence[IPv4Address],
+    island_port: int,
```

Comment:
Same as `ip_addresses` is not called `island_ip_addresses` it shouldn't be called `island_port`. This represents port services can reach the island, not necesarilly the island port

Reply:
By this logic, we should also not have it as an island port in the server config.

Reply:
We need a better name than "island_port" and "port" then. Any suggestions?

Reply:
> By this logic, we should also not have it as an island_port in the server config.

Server config is for users. Even better name there would be `server_port`, because it's more readable with the context users have

> We need a better name than "island_port" and "port" then. Any suggestions?

This represents the port the island can be accessed on. Either way we should stay consistent, so either rename the `ip_addresses` to `island_ip_addresses` or rename `island_port` to `port`